### PR TITLE
Changed get_course_info to count instead of fetch all.

### PR DESCRIPTION
**Pull request by moowiz**

### CODE REVIEW

**Comment by Sumukh** [comment]
Pending changes:

- This breaks the current sorting (lines 207 and 231) 
` all_backups.sort(key=lambda x, y: int(-5*(int(x.server_time > y.server_time) - 0.5)))`

- The frontend needs to adapt to the new changes.

**Comment by Some3141** [comment]
We really only care about sending the first 10 file contents, so if we can do this without reading everything, it's still an optimization. (Not sure if it'll be enough)

**Comment by Sumukh** [comment]
A few students can't use the web site at all because of this call timing out. It'd be great if we can get this working soon - we're pretty close!

**Comment by moowiz** [comment]
Does the frontend still need to adapt to these changes?

**Comment by moowiz** [comment]
@Some3141 can you shed some light on this sorting? I'm confused why we can't just use a [order](https://cloud.google.com/appengine/docs/python/ndb/queries#order) in the queries...

**Comment by Sumukh** [comment]
@moowiz  re: frontend - Yes - but it'll be much easier to do once the call works.

**Comment by Some3141** [comment]
@moowiz - I think I might not have realized I could order by time.

Also, is this ready to merge?

File: server/app/models.py [hunk 1]
```diff
@@ -202,10 +229,14 @@ def get_submissions(self, assignment, num_submissions=10):
         all_subms.sort(lambda x, y: int(-5*(int(x.server_time > y.server_time) - 0.5)))
 
         for subm in all_subms[:num_submissions]:
-            subm.messages = None
+            subm.messages = None # What is this here for???
```
Comment (Some3141) [inline server/app/models.py]: So we don't send the messages back to the server.

**Comment by Some3141** [comment]
Okay, everything looks good to me.

**Comment by Sumukh** [comment]
@Some3141 - This isn't ready to merge. You can test by starting the server and trying to view the dashboard. 

- [ ] This breaks the current sorting (lines 207 and 231) 
all_backups.sort(key=lambda x, y: int(-5*(int(x.server_time > y.server_time) - 0.5)))

- [ ] The frontend needs to adapt to the new changes.

**Comment by Sumukh** [comment]
LGTM.

**Pull request merged**